jupyter-extension/Dockerfile

```diff
@@ -31,3 +31,4 @@ USER $NB_UID
 COPY dist dist
 WORKDIR /home/jovyan
 RUN pip install `find /app/dist/ -name \*.whl` nbgitpuller
+RUN pip install determined
```

Comment:
You probably want to pin the version so it doesn't change out from under you when you rebuild

Reply:
Normally I'd agree, but I actually was thinking always grabbing latest might be best here?

Reply:
If this is just so that the users don't need to pip install determined then its probably fine, but as a result we won't be able to know what version of the determined SDK is included with a specified release of the extension without spinning up the container ourselves